src/howto.rst

```diff
@@ -814,3 +817,72 @@ There are variety of tools developed to caclulate free energy difference, includ
 
+Run a Multi-Sim
+---------------
+
+GOMC can automatically generate independent simulations with varying parameters from one input file.  
+This allows the user to sample a wider seach space.  To do so GOMC must be compiled in MPI mode, 
+and a couple of parameters must be added to the conf file.
+
+To compile in MPI mode, navigate to the GOMC/ directory and issue the following commands:
+
+.. code-block:: bash
+
+  $ chmod u+x metamakeMPI.sh
+  $ ./metamakeMPI.sh
+
+Then once the compilation is complete, set up the conf file as you would for a standard GOMC simulation.
+
+Finally, select one or more of the following parameters (``Temperature``, ``Pressure``, ``ChemPot``, ``Fugacity``) and enter more than one value for such parameters separated by a tab or space.
+
+  .. code-block:: text
+
+    #################################
+    # Mol.  Name Chem.  Pot.  (K)
+    #################################
+    ChemPot   ISB     -968     -974     -978     -982
+
+  .. code-block:: text
+
+    #################################
+    # Mol.  Name Chem.  Pot.  (K)
+    #################################
+    Fugacity   ISB     -968     -974     -978     -982
+
+  .. code-block:: text
+
+    #################################
+    # SIMULATION CONDITION
+    #################################
+    Temperature   270.00    280.00    290.00    300.00 
+
+  .. code-block:: text
+
+    #################################
+    # GEMC TYPE (DEFAULT IS NVT GEMC) 
+    #################################
+    GEMC        NPT
+    Pressure    5.76    5.80    5.84    5.88 
+
+.. Note:: GOMC will allow for more than one of these parameters (i.e. ChemPot and Temperature) to be greater than one, but the number of values given must either match between parameters or be one.  For example, a simulation with five chemPots must have either one temperature or five temperatures.  A simulation with five temperatures couldn't have three pressures.  This would cause GOMC to exit.
+
+A folder will be created for the output of each simulation, and the name will be generated from the parameters you choose. 
+A parent folder containing all the child folders will be created so as to not overpopulate the initial directory.  
+You may elect to choose the name of the folder in which the sub-folders for each replica are contained.
+Enter this name as a string following the ``MultiSimFolderName`` parameter.  If you don't provide this parameter, the default "MultiSimFolderName" will be used.
+
+  .. code-block:: text
+
+    MultiSimFolderName  outputFolderName
+
+
+.. Note:: To perform a multisim, GOMC must be compiled in MPI mode.  Also, if GOMC is compiled in MPI mode, a multisim must be performed.  To perform a standard simulation, use standard GOMC.
+
+
+The rest of the conf file should be similar to how you would set up a standard GOMC simulation.
+
+To initiate the multisimulation,
+
+    .. code-block:: bash
+
+       $ mpiexec -n #ofreplicas GOMC_xxx_yyyy <optional#ofthreads> conffile 
```

Comment:
Please add some explanation how GOMC will allocate the CPU for multiSim. How should user request number of nodes and threads or CPU to efficiently run MultiSim.

Reply:
You can use an example to explain that.

Reply:
![image](https://user-images.githubusercontent.com/39970712/72476016-1b519e80-37ba-11ea-9796-45a2439f14cb.png)

Reply:
![image](https://user-images.githubusercontent.com/39970712/72476135-6ec3ec80-37ba-11ea-9cbf-b7dd44c6068f.png)

Reply:
![image](https://user-images.githubusercontent.com/39970712/72476482-51435280-37bb-11ea-998d-81ed857970c0.png)